src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+  - [Prerequisites](#prerequisites)
+  - [Acquire an Access Token](#token)
+  - [Explore the API in JavaScript](#javascript)
+    - [Retrieve a User Access Token](#retrieve-token)
+    - [Post a Request](#post-request)
+
+
+### <a name="overview"></a>Overview
+
+The Request V4 API has 4 resources:
+
+1. __Expense resource__ - You can  select, create, update or delete an expense or show the list of expense link to a request.
+2. __Request resource__ - You can select, create, update or delete a request or show a list of requests.
+3. __Travel Agency resource__ - You can get the description of a travel agency office.
+4. __Workflow resource__ - You can advance a request in a new authorized status.
+
+Manage documents used for pre-spend authorizations within Concur Request.
+
+### <a name="version"></a>Version
+
+- 4.0
+- [3.0](/api-reference-deprecated/version-three/request/request.html)
+- [1.0](/api-reference-deprecated/version-one/Travel/travel-request.html)
+
+#### <a name="overviewV4"></a>Overview of Version 4.0
+
+Version 4.0 of the Requests API offers features like passive approval, Data retention, rigth to be forgotten.
+
+It works only with the new [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+### <a name="regionalavailability"></a>Regional Availability
+#### <a name="swagger"></a>With swagger
+For US production
+```
+https://sea-raas.concurasp.com
+```
+For EMEA production
+```
+https://par-raas.concurasp.com/
+```
+For China production
+```
+https://raas.service.cnqr.io/
+```
+#### <a name="curl"></a>With curl
+For US production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://us.api.concursolutions.com/travelrequest/v4/...
+```
+For EMEA production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://emea.api.concursolutions.com/travelrequest/v4/...
+```
+For China production
+```
+curl -X POST --H 'Content-Type: application/json' --H 'Accept: application/json' --header "Authorization: Bearer {YOUR ACCESS TOKEN}" -d { "name": "Your action name" } https://cn.api.concurdc.cn/travelrequest/v4/...
+```
+
+
+### <a name="explore-api"></a>Explore the API
+
+#### <a name="prerequisites"></a>Prerequisites
+
+1. [Create a sandbox](https://developer.concur.com/manage-apps/register.html) if you don't already have one.
+2. Read the [Getting Started](https://developer.concur.com/api-reference/authentication/getting-started.html) section of [Authentication API](https://developer.concur.com/api-reference/authentication/apidoc.html).
+
+Once you have registered your application, read about the [API endpoints](/api-reference/receipts/endpoints.html), or click the button to download a request collection for Postman.
```

Comment:
_note: Update the link [API endpoints] as it refers to the receipts endpoints and not the Request endpoints._

Reply:
FIXED